Week3/home-work/db.js

```diff
@@ -0,0 +1,40 @@
+const mysql = require('mysql');
+const util = require('util');
+const connection = mysql.createConnection({
+   host: 'localhost',
+   user: 'hyfuser',
+   password: 'hyfpassword',
+   database: 'todoapp'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function createData() {
+   let sql = "CREATE DATABASE IF NOT EXISTS todo_data"
+   let use_sql = "USE todo_data"
+   let users = `CREATE TABLE IF NOT EXISTS
+       Users(Id varchar(100) NOT NULL PRIMARY KEY UNIQUE,
```

Comment:
It is a good idea to make use of AUTO_INCREMENT here to help you along the way: https://dev.mysql.com/doc/refman/8.0/en/example-auto-increment.html